### DISCUSSION TITLE
Implementing Infinite Scroll with Firebase's startAfter

### DISCUSSION THREAD

**Discussion by iamalvisng** [reduxjs/redux-toolkit#1174] [Q&A]
Hi,

First of all, thanks for the new update with RTKQ! 

Understand that it is possible to use `queryFn` to query from Firebase, however I am puzzled on how to implement infinite scrolling with `startAt()` from Firebase by keeping track the last item from the previous query.

Do you have an example for such use case?

**Comment by phryneas** [comment]
I do not know firebase personally (you would have to give a lot more details, including code examples), but maybe already this thread on infinite scrolling in general can help you?

https://github.com/reduxjs/redux-toolkit/discussions/1163
